migration/transformation_schema_2_to_1.yaml

```diff
@@ -0,0 +1,134 @@
+# This file provides specific mappings for metric data between Java 2.x to Java 1.x
+# The format should be the same as the otel file_format for schema files.
+file_format: 1.1.0
+# TODO: Fix URL
+schema_url: https://github.com/signalfx/splunk-otel-java/migration/transformation_schema_2_to_1.yaml
+versions:
+  # Java version 2.0.0 is the new version. The following mappings translate
+  # FROM the new/correct/stable names back to the previous names.
+  # The names to use when applying attribute renaming are the NEW names.
+  # <new_name>: <old_name>
+  # We want to translate TO the old names as a migration step
+  2.0.0:
+    metrics:
+      changes:
+        #--------------------------------------------------------------
+        # Memory metrics
+        - rename_metrics:
+            jvm.memory.used:               runtime.jvm.memory.used           # micrometer
+            jvm.memory.committed:          runtime.jvm.memory.committed      # micrometer
+            jvm.memory.limit:              runtime.jvm.memory.max            # micrometer
+            jvm.memory.used_after_last_gc: runtime.jvm.memory.usage.after.gc # micrometer
```

Comment:
We're stating in the docs draft that there's no equivalent for this one. It is, in fact, red in the latest spreadsheet put together by Mateusz. Can we double check? Has anything changed?

Reply:
Well, the semantic convention might have solidified since then, I'm not sure. The one listed above is the comparable one, however the difference that you're probably pointing out is that the micrometer one is apparently a _percentage_ of total memory, whereas the otel one is in bytes.

So we have a choice -- do we map the micromter name to the new name so that people can use the old one in calculations, or do we just drop it from mappings so that the new units can be used with the new name?

Reply:
If folks keep using an old name but with different units, wouldn't they get a nasty surprise in their charts? It really depends on the principle we want to follow here. If we want to map everything at all costs, let's keep it as it is.

Would love to hear from @laurit, too.